packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -49,21 +49,30 @@ export const iconMetadata: {
     IconArrowPartialRotateLeft: {
         tags: ['reset']
     },
+    IconArrowRightThin: {
+        tags: ['link', 'open']
+    },
```

Comment:
Do we still need this icon since the link pattern has changed? I think the tags listed are incorrect now right @fredvisser ? What are the other use cases for keeping it?

Reply:
Yep - we still need it, because we added it to the [standalone link](https://stratus-storybook.ni.dev/iframe.html?viewMode=docs&id=patterns-links--docs&args=#standalone-links) pattern